index.d.ts

```diff
@@ -0,0 +1,40 @@
+export = prettier;
```

Comment:
There is already the [Prettier definition on DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/prettier/index.d.ts), and I do not think the TypeScript definition should be maintained here since not everyone knows how to write TypeScript, it's hard to tell contributors "You should also update the definition files".

Reply:
Oops! Sorry, couldn't find the module when installing `@types/prettier`. Turns out it was our internal npm mirror failing. Will remove.